.eslintrc.js

```diff
@@ -63,6 +63,16 @@ module.exports = {
     '@typescript-eslint/semi': ['warn'],
   },
   overrides: [
+    {
+      // Disable typescript rules for js files. This was occuring in
```

Comment:
this is odd... typescript rules should know not to act on js files

Reply:
:+1: Yeah... It looks like the rule itself does not check if the file is `.ts` or not. https://github.com/typescript-eslint/typescript-eslint/blob/v2.20.0/packages/eslint-plugin/docs/rules/explicit-function-return-type.md#configuring-in-a-mixed-jsts-codebase

Reply:
ah i see... at work we turn it off and then turn it on only for `.ts` files. which might be the better way, because semantically you want "do this rule for `.ts` files", not "do this rule for everything except `.js` files".